packages/library/src/providers/CFDProvider/CFDProvider.tsx

```diff
@@ -25,8 +27,9 @@ export const useCFDContext = () => {
     return context;
 };
 
-export const CFDProvider = ({ children }: { children: React.ReactNode }) => {
+export const CFDProvider = ({ children }: { children: ReactNode }) => {
     const [cfdState, setCfdState] = useState<TCFDState>({});
+    const [modalContent, setModalContent] = useState<ReactNode | null>(null);
```

Comment:
It is unnecessary to type `useState` unless when the bracket you have is empty example: `useState()`. I think its better to let it be like this:
```suggestion
    const [modalContent, setModalContent] = useState(null);
```
correct me if I'm wrong 😅 🙏🏼